src/main/generated/resources/data/minestuck/minestuck/dialogue/consort/book_purpose/buy_my_products.json

```diff
@@ -0,0 +1,40 @@
+{
+  "node": {
+    "messages": [
+      {
+        "type": "entity",
+        "message": "minestuck.dialogue.consort.book_purpose.buy_my_products.a"
+      },
+      {
+        "type": "entity",
+        "message": "minestuck.dialogue.consort.book_purpose.buy_my_products.b"
+      },
+      {
+        "type": "description",
+        "message": "minestuck.dialogue.consort.book_purpose.buy_my_products.description"
+      }
+    ],
+    "responses": [
+      {
+        "condition": {
+          "type": "minestuck:list",
+          "conditions": [
+            {
+              "type": "minestuck:always_true"
+            }
+          ],
+          "list_type": "none"
+        },
+        "fail_tooltip": "No matter how you feel, you are compelled to decline",
```

Comment:
Hang on, looks like there's a missing translation key here.

Reply:
Arnt normal fail tooltips untranslated?

Reply:
The condition-default tooltips are untranslated, but they shouldn't be. For response-specific tooltip like this, you can use `l.subText()` in the same way that you use `l.subMsg()` (except that arguments aren't supported).

Reply:
Alright, now done!